documentation/asciidoc/topics/ref_cluster_admin.adoc

```diff
@@ -0,0 +1,21 @@
+Log into your {k8s} cluster as an administrator.
```

Comment:
I'd go with simple list of requirements (prerequisites) and drop the commands: Adminstrator access, OKD 3.11 or Kubernetes x.xx.

We can assume that anyone who wants to install operator and have admin access to the platform probably knows how to login to OpenShift or Kuberentes.